src/psyclone/tests/test_files/dynamo0p3/infrastructure/field_mod.F90

```diff
@@ -719,7 +721,11 @@ subroutine log_minmax( self, log_level, label )
 
     write( log_scratch_space, '( A, A, A, 2E16.8 )' ) &
          "Min/max ", trim( label ),                   &
-         " = ", fmin%get_min(), fmax%get_max()
+    ! Functions get_min() and get_max() rely on MPI comms so
```

Comment:
@hiker's example calls `log_minmax`, however it fails to run due to the call to `mpi_mod` to enable MPI comms for calculations of the global minima and maxima. The `mpi_mod` will compile but most of the machinery is switched off so to not call MPI comms (I guess Joerg did this to mimimize changes to the code and still not allow MPI).
Since MPI is not supported, the minimum and maximum of a field is not calculated for each MPI process but just once from the field data. Does this make sense?